Makefile

```diff
@@ -12,6 +12,8 @@ run-agent-router-tests:
 	docker compose run --build backend poetry run pytest src/backend/tests/routers/test_agent.py
 run-user-router-tests:
 	docker compose run --build backend poetry run pytest src/backend/tests/routers/test_user.py
+run-chat-router-tests:
```

Comment:
Is there a reason we are adding new make commands instead of using the `run-tests` one with an arg?

Reply:
will remove, just convenient for local dev

Reply:
Thanks yeah, let's just remove those to avoid cluttering the makefile too much